parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -765,11 +778,6 @@ calcImports im tm = (im', render $ getUses result)
     -- further down, closer to the use sites.
     narrowestPossible :: Map Name (Prefix, Suffix, Int) -> Map Name (Prefix, Suffix, Int)
     narrowestPossible m = m |> Map.filter (\(p, s, i) -> not $ allInSubBlock tm p s i)
-    -- Don't do `use () ()` or `use Pair Pair`.  Tuple syntax generates ().() and Pair.Pair 
-    -- under the covers anyway.  This does mean that if someone is using Pair.Pair directly, 
-    -- then they'll miss out on FQN elision for that.  
-    exclusions :: Map Name (Prefix, Suffix, Int) -> Map Name (Prefix, Suffix, Int)
-    exclusions m = m |> Map.filterWithKey (\n _ -> notElem (Name.toString n) ["().()", "Pair.Pair"])

```

Comment:
There are plenty of other textual references to `"()"` and `"Pair"` in this file - do we need to get rid of those as well?